sites/fast-website/src/public/index.html

```diff
@@ -3,12 +3,114 @@
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
   <body>
     <fast-design-system-provider use-defaults>
-      <fast-button>Hello, World!</fast-button>
-      <gradient-picker>stub</gradient-picker>
+      <div class="wrapper">
+        <!--header-->
```

Comment:
nit: can these comments be like `<!--  section-name:start -->`, `<!-- section-name:end -->`